docs/porting/third-party-deps.md

```diff
@@ -0,0 +1,50 @@
+# Analyzing your 3rd Party Dependencies
+
+By [Phillip Carter](https://github.com/cartermp)
+
+The first step in the porting process is to understand your third party dependencies, figure out which of them (if any), don't run on .NET Core, and develop a contingency plan for those which don't run on .NET Core.
+
+## Prerequisites
+
+This article will assume you are using Windows and Visual Studio.  For your NuGet package dependencies, you may also wish to download the [Nuget Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).
+
+## Analyzing NuGet Packages
+
+Analyzing NuGet packages for portability is very easy.  Because a NuGet package is itself a set of folders which contain platform-specific assemblies, all you have to do is check to see if there is a folder which contains a .NET Core assembly.
+
+There are a few different folder names that are valid for .NET Core (these are also called Target Framework Monikers, or TFMs for short).  They are:
+
+1. `netstandard1x` or `netstandard1.x`, where `x` is a value from `0` to `5`.
+2. `dotnet5x` or `dotnet5.x`, where `x` is a value from `1` to `5`.
+3. `dnxcore50`
+
+Note that the latter options, `dotnet5x` and `dnxcore50`, are legacy TFMs from the early prerelease of .NET Core.  The vast majority of NuGet packages you encounter should be `netstandard`-based.
+
+Inspecting this is easiest with the [NuGet Package Explorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer).  Here's how to do it.
+
+1. Open the NuGet Package Explorer.
+2. Click "Open package from online feed".
+3. Search for the name of the package.
+4. Expand the "lib" folder on the right-hand side and look for a TFM that is valid for .NET Core.
+
+### What to do when your NuGet package dependency doesn't run on .NET Core
+
+Oh no!  There are a few things you can do if a NuGet package you depend on won't run on .NET Core.
```

Comment:
Oh no! seems very blog-like language. I think it can be removed from here without losing the meaning.